azure-pipelines.yml

```diff
@@ -156,25 +157,36 @@ jobs:
 
 - job: PackageWorkers
   dependsOn: ['Build_WINDOWS_X64', 'Build_WINDOWS_X86', 'Build_LINUX_X64', 'Build_OSX_X64']
-  condition: startsWith(variables['Build.SourceBranch'], 'refs/heads/release/')
+  condition: or(startsWith(variables['Build.SourceBranch'], 'refs/heads/release/'), eq(variables['Build.SourceBranch'], 'refs/heads/dev'))
   pool:
       vmImage: 'vs2017-win2016'
   steps:
   - bash: |
       echo "Releasing from $BUILD_SOURCEBRANCHNAME"
+      apt install jq
 
       if [[ $BUILD_SOURCEBRANCHNAME = 2\.* ]]
       then
+        echo "Generating V2 Release Package for $BUILD_SOURCEBRANCHNAME"
         NUSPEC="pack\Microsoft.Azure.Functions.V2.PythonWorker.nuspec"
+        WKVERSION="$BUILD_SOURCEBRANCHNAME"
       elif [[ $BUILD_SOURCEBRANCHNAME = 3\.* ]]
       then
+        echo "Generating V3 Release Package for $BUILD_SOURCEBRANCHNAME"
         NUSPEC="pack\Microsoft.Azure.Functions.V3.PythonWorker.nuspec"
+        WKVERSION="$BUILD_SOURCEBRANCHNAME"
+      elif [[ $BUILD_SOURCEBRANCHNAME = dev ]]
+      then
+        echo "Generating V3 Integration Test Package for $BUILD_SOURCEBRANCHNAME"
+        LATEST_TAG=$(curl https://api.github.com/repos/Azure/azure-functions-python-worker/tags -s | jq '.[0].name' | sed 's/\"//g')
+        NUSPEC="pack\Microsoft.Azure.Functions.V3.PythonWorker.nuspec"
+        WKVERSION="3.$LATEST_TAG-dev"  # Integration Test (e.g. 3.1.1.6-dev)
```

Comment:
So - what this PR is doing is to greater a new nuspec package called `3.$LATEST_TAG-dev`, but not actually create a release on GitHub. Correct?

Reply:
Yes, it does not create a release. Only an artifact in https://azfunc.visualstudio.com/Azure%20Functions/_packaging?_a=feed&feed=AzureFunctionsPreRelease